### PR TITLE
[SDK-9699] Add Basic TV Player BPA

### PR DESCRIPTION
**Pull request by david-almaguer**
Adds a basic player implementation using JWPlayerTVKit tvOS SDK 4.x.x.
- Global README.md is updated.
- Added new groups to separate iOS targets from tvOS targets.
- Added new BasicTVPlayer target

### CODE REVIEW

File: JWBestPracticeApps/Podfile [hunk 1]
```diff
@@ -79,3 +83,16 @@ target 'Picture in Picture' do
   project 'Picture in Picture/Picture in Picture'
   common_JWPlayer
 end
+
+####################################
+#### JWPlayerTVKit tvOS targets ####
+####################################
+
+def common_JWPlayerTVKit
+    pod 'JWPlayerTVKit', '~> 1'
```
Comment (DavidPerezP124) [inline JWBestPracticeApps/Podfile:92]: It seems we need to specify the platform for tvOS as well. We can let the `pod install` script decide it automatically by removing the first line but it throws a few warnings for this.

**Review by DavidPerezP124 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
 # JWPlayerKit 4.x Best Practice Apps
 
-This repository contains samples relating to the JWPlayerKit SDK version 4.x for iOS. 
+This repository contains samples relating to the JWPlayerKit SDK version 4.x for iOS and JWPlayerTVKit SDK version 4.x for tvOS.
```
Comment (s-seibert) [inline README.md:3]: Shouldn't this say 1.x for tvOS?

**Review by s-seibert — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (david-almaguer) [reply README.md]: Addressed

**Review by s-seibert — APPROVED** [review]

**Review by Tozarf — APPROVED** [review]
LGTM 💯

**Pull request merged**